packages/light/lib/index.js

```diff
@@ -1,41 +1,34 @@
+// Copyright (c) Jupyter Accessibility Team.
```

Comment:
This folder will change in future PRs, the `light` and `dark` folders will be replaced with each one of the themes that we are going to add to this extension. Linting will be done on those PRs once they are opened.

Reply:
If this is basically a setup PR why not adding linters and pre-commits here rather than mixing with PRs aimed at development? 
That ensures we have all the boilerplate we need from the start